components/CounterScreen.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { MaterialTopTabNavigationProp } from '@react-navigation/material-top-tabs';
+import MatterhornApiClient, { CounterResponse } from '../api-client/MatterhornApiClient';
+import { RootNavParamList } from '../App';
+
+import getEnvVars from '../environment';
+import PocButton from './PocButton';
+
+const { apiUrl } = getEnvVars();
+const apiClient: MatterhornApiClient = new MatterhornApiClient(apiUrl);
+
+const styles = StyleSheet.create({
+  container: {
+    flex: 1,
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center'
+  }
+});
+
+function retrieveCount(api: MatterhornApiClient, setCount: React.Dispatch<React.SetStateAction<number>>) {
+  api.Counter().then((value: CounterResponse) => {
+    setCount(value.Value);
+  }).catch((reason) => {
+    console.log('Request failed!');
+    console.log(reason);
+  });
+}
+
+type CounterScreenNavigationProp = MaterialTopTabNavigationProp<
```

Comment:
is this necessary? do these props actually get passed to and used by the component?

Reply:
Looks like it isn't actually necessary unless we want to be able to use the navigation object from within the screen components. I'll remove.